pipelines/extension-build.yml

```diff
@@ -18,36 +15,83 @@ pr:
     - /extensions/
     - /pipelines/extension-build.yml
 
-steps:
-- task: NPM@1
-  displayName: npm install
-  inputs:
-    command: install
-    workingDir: extensions
-
-- task: NPM@1
-  displayName: npm run lint
-  inputs:
-    command: custom
-    customCommand: run lint
-    workingDir: extensions
-
-- task: NPM@1
-  displayName: npm run build
-  inputs:
-    command: custom
-    customCommand: run build
-    workingDir: extensions
-
-- task: CopyFiles@2
-  displayName: Stage extension artifact for publish
-  inputs:
-    sourceFolder: extensions
-    contents: '**/*.vsix'
-    targetFolder: $(Build.ArtifactStagingDirectory)
-
-- task: PublishPipelineArtifact@0
-  displayName: Publish extension artifact
-  inputs:
-    artifactName: drop
-    targetPath: $(Build.ArtifactStagingDirectory)
+stages:
+- stage: Build
+  jobs:
+  - job: buildvsix
```

Comment:
Is there a requirement that the job name has to be [a-z0-9]+ or can we use a more human readable name here?

Reply:
name of the job, A-Z, a-z, 0-9, and underscore from https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#job

Reply:
done